packages/@lwc/engine/src/framework/def.ts

```diff
@@ -279,6 +286,7 @@ import {
     DecoratorMeta,
     PropsDef,
 } from './decorators/register';
+import { defaultEmptyTemplate } from './secure-template';
```

Comment:
Any reason for these imports are not at the top of the file?

Reply:
old stuff probably! trying to mitigate circular references. we can update that at some point.